search-engines.lisp

```diff
@@ -1074,6 +1074,130 @@ REQUEST-ARGS is a list of args to pass to request function."
      :completion-function (make-pkgs-completion)
      :documentation "UNIX/GNU/Linux package search."))
 
+(define-search-engine peertube
+    (:shortcut "pt"
+     :fallback-url (quri:uri "search.joinpeertube.org")
+     :base-search-url "https:///search.joinpeertube.org/search?search=~a"
+     :documentation "Peertube `nx:search-engine' via its Sepia Search global search engine.")
+  (sort-by "sort" ((:best "-match")
+                   (:newest "-publishedAt")
+                   (:oldest "publishedAt")))
+  (published-date "publishedDateRange" ((:any "any_published_date")
+                                        (:today "today")
+                                        (:month "last_month")
+                                        (:week "last_7days")
+                                        (:year "last_365days")))
+  (is-live "isLive" ((nil "false")
+                     (t "both")))
+  (nsfw "nsfw" ((nil "false")
+                (t "both")))
+  (duration "durationRange" ((:any "any_duration")
+                             (:short "short")
+                             (:medium "medium")
+                             (:long "long")))
+  (category "categoryOneOf" ((:all "")
+                             (:music "1")
+                             (:films "2")
+                             (:vehicles "3")
+                             (:art "4")
+                             (:sports "5")
+                             (:travels "6")
+                             (:gaming "7")
+                             (:people "8")
+                             (:comedy "9")
+                             (:entertainment "10")
+                             (:news-and-politics "11")
+                             (:how-to "12")
+                             (:education "13")
+                             (:activism "14")
+                             (:science-and-technology "15")
+                             (:animals "16")
+                             (:kids "17")
+                             (:food "18")))
+  (licence "licenceOneOf" ((:all "")
+                           (:by "1")
+                           (:by-sa "2")
+                           (:by-nd "3")
+                           (:by-nc "4")
+                           (:by-nc-sa "5")
+                           (:by-nc-nd "6")
+                           (:cc0 "7")))
+  (language "languageOneOf" ((:all "")
```

Comment:
Consider adding short country codes, like `en' or `fr'. It's not a requirement, though, feel free to ignore.